js/tests/README.md

```diff
@@ -0,0 +1,44 @@
+## How does Bootstrap's test suite work?
+
+Bootstrap uses [QUnit](http://api.qunitjs.com/), a powerful, easy-to-use JavaScript unit test framework. Each plugin has a file dedicated to its tests in `unit/<plugin-name>.js`.
+
+* `unit/` contains the unit test files for each Bootstrap plugin.
+* `vendor/` contains third-party testing-related code for the browser environment (jQuery and QUnit).
```

Comment:
These files are also used when testing in PhantomJS so the remark about the browser environment isn't accurate.